src/styles/_common.scss

```diff
@@ -127,6 +127,10 @@
   display: contents;
 }
 
+.uppy-Dashboard-AddFiles-info {
+  display: inline-block;
+}
+
```

Comment:
This does not belong to common stuff. If this is the first case of overriding something that already exists, create _overrides.scss partial and throw it there.